app/dashboard/models.py

```diff
@@ -1668,7 +1668,6 @@ class Profile(SuperModel):
     user = models.OneToOneField(User, on_delete=models.SET_NULL, null=True, blank=True)
     data = JSONField()
     handle = models.CharField(max_length=255, db_index=True, unique=True)
-    avatar = models.ForeignKey('avatar.Avatar', on_delete=models.SET_NULL, null=True, blank=True)
```

Comment:
I don't understand this move.  Why are we flipping the relation?  Can't you access `profile.avatar` **or** `avatar.profile_set` ?

Reply:
`profile.avatar` gives me single avatar
`avatar.profile_set` gives me all profiles with this avatar
Can one avatar be shared among profiles ?

I think we need to model profile with multiple avatars - that's why we are flipping relation.
How would I retrieve 'My avatars' without this move?